WPFSKillTree/Model/Bandit.cs

```diff
@@ -8,14 +8,14 @@ namespace POESKillTree.Model
 {
     /// <summary>
     /// Enumeration of the bandits from whom can be choose at each difficulty.
-    /// (chosing none gives an extra skill point)
+    /// (choosing none gives an extra skill point)
     /// </summary>
     public enum Bandit
     {
-        None,
-        Alira,
-        Oak,
-        Kraityn
+        None = 0,
+        Alira = 1,
+        Oak = 2,
+        Kraityn = 3
```

Comment:
I've tried it during implementation and it worked. I've searched all occurrences manually, and they looked fine. I've wrote some odd tests and they passed. But I'm still not sure that I can oversee everything, e.g. there might be some old *.pbuild files that were deserialized using numeric values or other xml/json (code that created compatible content but now burried in repo history) or something else, that I'm not aware of.
Moreover, some day we can get another great online planner with it's own format, or update for the official planner which values would be more preferable for us to use I believe.
Keeping all that in mind I've left values as is, but if you think this is overhead I'll remove whole `BanditConverter` and will just switch values.